packages/elements/package.json

```diff
@@ -19,7 +19,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/stryker-mutator/mutation-testing-elements.git"
+    "url": "git+https://github.com/stryker-mutator/mutation-testing-elements.git",
```

Comment:
I think we should remove the`git+` prefix while we're at it. I'm pretty sure this isn't a valid URL, which might be a problem for renovate bot.

Reply:
I also see this in the root `package.json`, should I remove this one as well?

Reply:
Actually renovatebot doesn't look at `repository` much for grouping PR's. It's done hardcoded: https://github.com/renovatebot/renovate/pull/21905

Reply:
This blows my mind! 🤯

Reply:
This blows my mind! 🤯